makefile.go

```diff
@@ -0,0 +1,220 @@
+package main
+
+import (
+	"bytes"
+	"flag"
+	"fmt"
+	"log"
+	"os"
+	"os/exec"
+	"path"
+	"path/filepath"
+)
+
+var (
+	KUBE_GO_PACKAGE string
+	K8SM_GO_PACKAGE string
+	K8_CMD          []string
+	tempDirs        []string
+	CMD_DIRS        []string
+	LIB_DIRS        []string
+	target          = "./target"
+)
+
+func run(dir, command string, args ...string) (output string, err error) {
```

Comment:
the output string is never actually used anywhere, should probably kill it and just return error